src/process/qgsprocess.cpp

```diff
@@ -298,6 +299,17 @@ int QgsProcessingExec::run( const QStringList &constArgs )
     listAlgorithms( useJson );
     return 0;
   }
+  else if ( command == QLatin1String( "--version" ) || command == QLatin1String( "-v" ) )
+  {
+    const QString msg = QStringLiteral( "QGIS %1 '%2' (%3)\n" ).arg( VERSION, RELEASE_NAME, QGSVERSION );
+    std::cout << msg.toStdString();
+    return 0;
+  }
+  else if ( command == QLatin1String( "--help" ) || command == QLatin1String( "-h" ) )
+  {
+    showUsage( args.at( 0 ) );
+    return 0;
+  }
```

Comment:
Should these two new option be added to the "Usage"?

Reply:
Yes indeed!